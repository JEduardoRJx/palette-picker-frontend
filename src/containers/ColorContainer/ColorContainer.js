import React from 'react';
import { ColorCard } from '../ColorCard/ColorCard';
import './ColorContainer.scss';

export const ColorContainer = ({ colors, toggleLock }) => {
    const allColorCards = colors.map(color => {
    return <ColorCard 
    colorInfo={color} 
    toggleLock={toggleLock}
    key={color.color} />
  })

  return (
    <section className="color-container">
      {allColorCards}
    </section>
  )
}

export default ColorContainer;