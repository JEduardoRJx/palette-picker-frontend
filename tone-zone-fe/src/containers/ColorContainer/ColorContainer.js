import React from 'react';

export const ColorContainer = () => {
  return (
    <section className="color-container">
      
    </section>
  )
}

export default ColorContainer;