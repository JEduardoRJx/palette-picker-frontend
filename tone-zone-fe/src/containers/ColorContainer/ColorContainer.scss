.color-container {
  display: flex;
  margin-bottom: 0px;
}