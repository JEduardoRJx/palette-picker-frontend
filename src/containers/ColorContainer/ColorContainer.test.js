import React from 'react';
import { shallow } from 'enzyme';
import ColorContainer from './ColorContainer'

describe('ColorContainer', () => {
  const mockColor = [{
    color: "#cecccc",
    isLocked: false
  }]

  it('should match the snapshot ', () => {
    const wrapper = shallow(<ColorContainer
      colors={mockColor} 
      toggleLock={jest.fn()}
    />);
    expect(wrapper).toMatchSnapshot();
  });
});