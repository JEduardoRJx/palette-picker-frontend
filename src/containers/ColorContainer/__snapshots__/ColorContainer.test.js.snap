// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColorContainer should match the snapshot  1`] = `
<section
  className="color-container"
>
  <ColorCard
    colorInfo={
      Object {
        "color": "#cecccc",
        "isLocked": false,
      }
    }
    key="#cecccc"
    toggleLock={[MockFunction]}
  />
</section>
`;
